docs/Installation/Installation.md

```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
 
 \tableofcontents
 
-This page details the installation procedure for SpECTRE on personal computers
-using x86-64 processors. For configuring SpECTRE, please refer to the
-\subpage spectre_build_system page. For instructions on installing SpECTRE on
-clusters please refer to the \subpage installation_on_clusters page. For
-instructions on installing SpECTRE on Apple Silicon Macs, please refer to the
-\subpage installation_on_apple_silicon page. Refer to the
-\subpage versioning_and_releases page for information on specific versions
-to install.
+### Running containerized releases
+
+A quick way to run the code without installing anything at all is with our
+containerized releases:
```

Comment:
I think we need to emphasize here that the entry point to this container is the CLI (and link to the docs) and that only very specific executables are built, namely SolveXcts, BBH, SingleBH. We don't want to give people the impression that they can just download this container and run anything from spectre.

Reply:
Currently only CLI, CCE, and SolveXcts are built